bin/link-all.mts

```diff
@@ -55,7 +57,7 @@ const unpack = pkgs.map(async (pkg) => {
     const packageJsonPath = join(pkgDest, 'package.json');
     const packageJson = JSON.parse(await readFile(packageJsonPath, { encoding: 'utf8' }));
     delete packageJson.devDependencies;
-    await writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2), {
+    writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2), {
```

Comment:
`writeFileSync` is not async, so no need for `await`. There are a number of cases of this in the PR.